src/Service.Tests/Authorization/GraphQL/GraphQLMutationAuthorizationUnitTests.cs

```diff
@@ -40,9 +40,14 @@ public void AuthorizeDirectiveAddedForMutation(EntityActionOperation operationTy
                 ";
 
             DocumentNode root = Utf8GraphQLParser.Parse(gql);
+            Dictionary<string, DatabaseType> entityNameToDatabasetype = new()
+            {
+                { "Foo", DatabaseType.MSSQL }
```

Comment:
nit: define "Foo" as const string entityName, and reuse whereever its used in this function

Reply:
fixed.